tests/utils/bits.py

```diff
@@ -0,0 +1,7 @@
+def to_uint(a):
+    a = int(a)
+    return (a & ((1 << 128) - 1), a >> 128)
+
+
+def combine_ints(low: int, high: int) -> int:
+    return (high << 128) + low
```

Comment:
there are already utils for this, see https://github.com/sayajin-labs/kakarot/blob/main/tests/integration/helpers/helpers.py